projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -162,11 +163,38 @@ public function callback( $path = '', $blog_id = 0 ) {
 
 		remove_filter( 'user_search_columns', array( $this, 'api_user_override_search_columns' ) );
 
+		$is_wpcom        = defined( 'IS_WPCOM' ) && IS_WPCOM;
+		$include_viewers = (bool) $args['include_viewers'] && $is_wpcom;
```

Comment:
You may want to add some checks here and below to avoid PHP notices for parameters that are not set (like when doing a query without setting the `include_viewers` param):

```
[18-Jan-2023 17:30:55 UTC] PHP Notice:  Undefined index: include_viewers in /wp-content/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php on line 167
```

Reply:
I've fixed this for the `include_viewers` argument. I'm unsure if it's wise to go and fix this for every other argument in this function (in this PR) as it can happen to all of them. Thanks for the find!